### PR TITLE
Add a basic text reftest

### PR DESCRIPTION
**Pull request by jrmuizel**
This change is [Reviewable](https://reviewable.io/reviews/servo/webrender/1070)

### CODE REVIEW

File: wrench/reftests/text/blank.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+root:
+  items:
+        - type: rect
+          bounds: [0, 0, 95, 88]
```
Comment (rlhunt) [inline wrench/reftests/text/blank.yaml]: Does this need to have a blank rect?

Comment (jrmuizel) [inline wrench/reftests/text/blank.yaml]: I believe we panic if items is empty. I didn't investigate what was happening.

Comment (rlhunt) [inline wrench/reftests/text/blank.yaml]: Oh yes, that does happen. `reftest/gradients/blank.yaml` gets around that by not specifying a `items:`
```yaml
---
root:
```

**Pull request merged**